website/docs/build-apps/how-to-guides/import-export-app-entities.md

```diff
@@ -0,0 +1,33 @@
+---
+description: A guide that outlines steps on how to export entities from a page to a JSON file and subsequently import it into another page.
+---
+
+# Reuse App Entities
+
+When working on complex Appsmith applications, you might find the need to reuse a particular set of widgets, queries, JSObjects, and more across pages. Instead of recreating these entities from scratch each time, Appsmith provides a way for you to export entities from one page and import them into another. This guide outlines steps on how to export entities from a page to a JSON file and subsequently import it into another page.
+
+## Export entities
+
+Follow these steps to export your desired entities:
+
+* Open the Appsmith editor and access the app containing the page you wish to export.
```

Comment:
i think this step is unecessary

```suggestion
```

Reply:
This is needed if they want to change the app. Will try to reword.